src/mixxxapplication.cpp

```diff
@@ -117,6 +122,13 @@ bool MixxxApplication::notify(QObject* target, QEvent* event) {
             mouseEvent->setButton(Qt::RightButton);
             m_rightPressedButtons++;
         }
+        if (mouseEvent->button() == Qt::RightButton && mouseEvent->buttons() == Qt::LeftButton) {
```

Comment:
Workarounds for specific Qt releases should be enclosed in #ifef guards and restricted to only the Qt versions that are actually affected.

Reply:
got it. it also also platform specific, so i will guard that too.

Reply:
If you search the code you will find some common patterns involving `QT_VERSION_CHECK` etc.